src/app/news/page.tsx

```diff
@@ -1,15 +1,52 @@
-import { POSTS } from '@/data/posts'
-import { formatDistance } from 'date-fns'
+'use client'
+
+import { useState } from 'react'
+import NewsCard from '@/components/NewsCard'
+import { NewsLine, NewsSection } from '@/components/Section'
+import Post, { POSTS } from '@/data/posts'
+import NewsModal from '@/components/NewsModal'
 
 export default function Page() {
+  const [modal, setModal] = useState(false)
```

Comment:
do we need this variable? isn't `modalContent === null equivalent` to `modal === false`?

Reply:
I removed this in the [second commit](https://github.com/kixlab/website/pull/15/commits/e49beb647bb716fcaafc5caaae734ff75d5c005b)